core/api/pod-api/src/default.ts

```diff
@@ -110,7 +110,7 @@ export class DefaultPod implements Pod {
                 return fs.stat(path);
             }
 
-            writeFile(path: string, content: string, options: EncodingOptions): Promise<void> {
+            writeFile(path: string, content: string, options?: EncodingOptions): Promise<string> {
```

Comment:
Are you sure this is not `void`, `null` or `undefined`? [Here says *fulfills with undefined upon success*](https://nodejs.org/api/fs.html#fspromiseswritefilefile-data-options)